src/connectors/hits/__tests__/connectHits-test.js

```diff
@@ -80,8 +105,31 @@ describe('connectHits', () => {
       createURL: () => '#',
     });
 
+    const processedHits = [
+      {fake: 'data'},
+      {sample: 'infos'},
+      {
+        toEscape: '&lt;a href=&quot;#top&quot;&gt;Go to top&lt;/a&gt;',
+        _highlightResult: {
+          toEscape: {
+            wontEscape: '<h1>Not escaped</h1>',
+            value: '&lt;a href=&quot;#top&quot;&gt;<em>Go to top</em>&lt;/a&gt;',
+          },
+        },
+      },
+      {
+        whitelisted: '<a href="#top">Go to top</a>',
+        _highlightResult: {
+          whitelisted: {
+            wontEscape: '<h1>Not escaped</h1>',
+            value: '<a href="#top"><em>Go to top</em></a>',
+          },
+        },
+      },
+    ];
+
     const secondRenderingOptions = rendering.lastCall.args[0];
-    expect(secondRenderingOptions.hits).toEqual(hits);
+    expect(secondRenderingOptions.hits).toEqual(processedHits);
```

Comment:
We should probably add tests without the escape mechanism and one that tests the white list option.